### PR TITLE
Add back in volume claim template

### PR DESCRIPTION
**Pull request by moosecanswim**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

This pr adds back in the volumeclaimetemplate that is required to perform an upgrade from 1.2 -> 1.3.

2. What **changes to custom.yaml** are required?

none

3. Have you documented any additional setup steps or configurations options?

none needed

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

run `make install` with the release-2.2 branch.  then switch to this branch, cd into fabric and run `make upgrade-fabric`

### CODE REVIEW

File: fabric/control-api/values.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@ controlApi:
     gm_control_api_redis_retry_delay:
       type: value
       value: '{{- if .Values.global.external_redis.disabled }}5s{{- else}}{{ .Values.global.external_redis.retry_delay }}{{- end }}'
-  resources:
+  # resources:
```
Comment (chrisbsmith) [inline fabric/control-api/values.yaml:142]: We should really define resources for our services.  We don't have to do it here, but let's add an issue to do it.

**Comment by moosecanswim** [comment]
This is failing because the edge node is not accessable because of the jwt issue that is being addressed in the 1.3 branch

**Review by chrisbsmith — APPROVED** [review]
LGTM

**Review by chrisbsmith — DISMISSED** [review]
LGTM

**Review by chrisbsmith — COMMENTED** [review]
@moosecanswim, there's an issue with the data versioning.  Can you fix that with this PR? 

https://github.com/greymatter-io/helm-charts/runs/1408566591?check_suite_focus=true

**Review by chrisbsmith — APPROVED** [review]
LGTM

**Pull request merged**